### ISSUE TITLE
The units of the timeline cannot be set using the keyboard

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#4470]
### Steps to reproduce

Using the keyboard, try to set the units of the timeline to beats and measures

### Expected behavior

It's possible

### Actual behavior

It's not possible

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

This can be done in Reaper, using a sub menu on the View menu.

**Comment by petersampsonaudacity** [comment]
Tested this on W10 with latest 3.3.0 Beta and this works fine from the Extra menu.

I have updated the Manual for this.
